quality/detekt.yml

```diff
@@ -252,8 +228,6 @@ style:
   MethodNameEqualsClassName:
     active: false
     ignoreOverriddenFunction: true
-  ModifierOrder:
-    active: true
```

Comment:
I think we should leave this one on: while I don't care about all the modifier orders, I really want `private/public/etc.` on the outside.